app/assets/stylesheets/plug/trix-overrides.scss

```diff
@@ -0,0 +1,8 @@
+.trix-button-row {
+  overflow: hidden;
+}
+
+.trix-button--icon-code,
+.trix-button-group--file-tools {
```

Comment:
hey bro,
 I think your editor can still allow the drag and drop the file. you might need to remove `trix-file-accept`
```

  document.addEventListener('trix-file-accept', function(event) {
    event.preventDefault();
  });

```

Reply:
Good point @yarhtut. Done.